Evergreen/Apps/Get-McNeelRhino.ps1

```diff
@@ -0,0 +1,57 @@
+Function Get-McNeelRhino {
+    <#
+        .SYNOPSIS
+            Get the current version and download URIs for the supported releases of Rhino.
+
+        .NOTES
+            Author: Andrew Cooper
+            Twitter: @adotcoop
+    #>
+    [OutputType([System.Management.Automation.PSObject])]
+    [CmdletBinding(SupportsShouldProcess = $False)]
+    param (
+        [Parameter(Mandatory = $False, Position = 0)]
+        [ValidateNotNull()]
+        [System.Management.Automation.PSObject]
+        $res = (Get-FunctionResource -AppName ("$($MyInvocation.MyCommand)".Split("-"))[1]),
+
+        [Parameter(Mandatory = $False, Position = 1)]
+        [ValidateNotNull()]
+        [System.String] $Filter
+    )
+
+    foreach ($Release in $res.Get.Releases.GetEnumerator()) {
+
+        # Query the Rhino update API
+        $UpdateFeed = Invoke-WebRequestWrapper $Release.Value
+
+        If ($Null -ne $UpdateFeed) {
+
+            # Convert response from UTF8
+            Try { 
```

Comment:
`Invoke-RestMethodWrapper` might avoid the need to convert encoding and into XML